contributing_to_docs/doc_guidelines.adoc

```diff
@@ -381,6 +381,15 @@ include::modules/module-filename.adoc[leveloffset=+1] <1>
 <1> `[leveloffset=]` link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-with-leveloffset/#manipulate-heading-levels-with-leveloffset[is relative], and changes the header level of included content.
 
 You can use `tag` attributes to link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions[define regions of a file and include by tag instead of the whole file.] This enables you to make conditional modifications to existing modules instead of duplicating content. When creating a tag, use a tag name that makes it easy to understand where the content defined will be included.
++
+[WARNING]
+====
+Some filtering functionality of tagged regions are not fully supported by PV1. Use the following with caution or not at all. Verify that access.redhat.com builds and docs.openshift.com builds are including or excluding content as expected:
+
+* The wildcard (*) is not supported. Do not use.
+* Use the `tags` (plural) attribute when there is more than one tag specified. Use the `tag` (singular) attruibte when there is only one tag.
+* Use an escape with tag attributes that are in YAML blocks. 
```

Comment:
I'm either misunderstanding, or this is redundant. Per the asciidoc documentation: "tag directives will be placed after a line comment as defined by the language of the source file" a YAML line comment is a #. An escape, in Asciidoc is a `/` and would render a tag null, as tags are just attributes.